tests/Rector/Class_/CommandPropertyToAttributeRector/Fixture/commandWithAttributeNameSetButDescriptionNot.php.inc

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Should fixture file names be named `like_this.php.inc` or `likeThis.php.inc`?

Reply:
`like_this.php.inc`

Reply:
I updated the test fixture file names for CommandPropertyToAttributeRector